concepts/05 UI Components/Scheduler/040 Resources/030 Group Appointments by Resources.md

```diff
@@ -1,4 +1,4 @@
-To group appointments by resources, assign an array to the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') property. Each element of this array is **fieldExpr** of a resource kind. Note that the resource headers' order depends on the resources' order in the [resources](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/') array.
+To group appointments by resources, assign an array to the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') property. Each element of this array is **fieldExpr** of a resource kind. Note that the resource headers' order depends on the resources' order in the [resources](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/') array. The Scheduler does not support grouping resources by multiple data fields in the **agenda** view.
 
 ---
 
```

Comment:
Please make the following change below:

Line 201:
"You can change resource headers' orientation in an individual view using..."

Please change "headers' orientation" to "header orientation".

Reply:
Done